components/service/glean/src/test/java/mozilla/components/service/glean/TestUtil.kt

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean
+
+import org.json.JSONObject
+
+fun checkPingSchema(content: JSONObject) {
+    val proc = ProcessBuilder(
+        listOf("./build/bootstrap/Miniconda3/bin/python", "-m", "glean_parser", "check")
```

Comment:
I wonder if there's a better way to do this, without hardcoding paths. Maybe adding this to the resources? @pocmo , thoughts?

Reply:
With those paths being used in the gradle scripts and the tests I can't think of a shared place where both world have access. From the gradle script you could inject it into a constant; but that constant would be available to all glean code and not limited to tests.